R/ImportMethods.R

```diff
@@ -982,3 +985,78 @@ setGeneric("importPublicData",
 
 setMethod("importPublicData", signature(origin = "character", dataset = "character", sample = "character"),
           .importPublicData)
+
+
+#' Read in BigWig files to CAGEexp object
+#'
+#' @param genome the name of the reference genome (bsgenome)
+#' @param filepath list of input bigwig files with full path
+#' The filenames should include a "_str1" and "_str2" substring after the sample name
+#' The sampe name will be the prefix before these substrings
+#' the filepath in the CAGEexp object will only include the bw corresponding to str1
+#' 
+#' @return a CAGEexp object
+#' 
+#' @author Katalin Ferenc
+#' @author Damir Baranasic
+#' 
+#' @examples
+#' filePaths <- c(
+#' "data/NCig10061_subsampled_str1.Signal.Unique.str1.out.wig.bw",
+#' "data/NCig10061_subsampled_str2.Signal.Unique.str2.out.wig.bw")
+#' import.bigwig(
+#'  genome="BSgenome.Drerio.UCSC.danRer7",
+#'  filepath=filePaths
+#'  )
+
+import.bigwig <- function(
+    genome,
+    filepath){
+
+    signals = lapply(
+        filepath,
+        function(x) {
+            track_in <- rtracklayer::import(x)
+            coerceInBSgenome(track_in, genome)
```

Comment:
This will run later in https://github.com/charles-plessy/CAGEr/blob/devel/R/ImportMethods.R#L586, so you do not need it in this import function.